include/crocoddyl/multibody/residuals/contact-cop-position.hxx

```diff
@@ -14,12 +14,22 @@ template <typename _Scalar>
 ResidualModelContactCoPPositionTpl<_Scalar>::ResidualModelContactCoPPositionTpl(
     boost::shared_ptr<StateMultibody> state, const pinocchio::FrameIndex id, const CoPSupport& cref,
     const std::size_t nu)
-    : Base(state, 4, nu, true, true, true), id_(id), cref_(cref) {}
+    : Base(state, 4, nu, true, true, true), id_(id), cref_(cref) {
+  if (static_cast<pinocchio::FrameIndex>(state->get_pinocchio()->nframes) >= id) {
```

Comment:
I believe this should be the other way round, `<=`

Reply:
Yes, I have (force) pushed the right condition